docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
   auth-service:
     container_name: as_auth-service
     image: mojaloop/auth-service:local
+    command: sh -c "/opt/auth-service/scripts/wait4.js my-sql"
```

Comment:
Don't you want to run the server after completing this command? Take a look at `pisp/docker-local/docker-compose.yml`